docs/installation.md

```diff
@@ -1,19 +1,93 @@
-# Installation Guide
+# Installation Cyclone
 
-## From Source Code
+## Install With Helm
 
-### Build and push images
+The simplest way to install Cyclone is using Helm chart. By default, images are pulled from DockerHub, so make sure DockerHub is accessible from your cluster.
 
-Here <registry>/<project> specifies the registry and project where to push your images, for example, test.caicloudprivatetest.com/release.
+```bash
+$ helm install ./helm/cyclone
+```
+
+If you want to use your own private registry, you can configure it as:
+
+```bash
+$ helm install --set imageRegistry.registry=cargo.caicloud.xyz,imageRegistry.project=release ./helm/cyclone
+```
+
+For more detailed configuration, please use values file, same format to [default values file](../helm/cyclone/values.yaml).
```

Comment:
I think this is to recommend values file for more configurations, but it is not clear.